### ISSUE TITLE
Inlining throws NullPointerException

### ISSUE THREAD

**Issue by OliverSch** [OliverSch/compiler#102]
```
===TESTCASE===
Simon_150
---CUSTOM---
Exception in thread "main" java.lang.NullPointerException
	at edu.kit.pp.mjcompiler.mjcompiler.firm.optimize.MethodInline.inline(MethodInline.java:90)
	at edu.kit.pp.mjcompiler.mjcompiler.firm.optimize.MethodInline.perform(MethodInline.java:51)
	at edu.kit.pp.mjcompiler.mjcompiler.firm.FirmProgram.optimize(FirmProgram.java:137)
	at edu.kit.pp.mjcompiler.CompileCommand.generateOptimizedCode(CompileCommand.java:66)
	at edu.kit.pp.mjcompiler.CompileCommand.execute(CompileCommand.java:48)
	at edu.kit.pp.mjcompiler.CompileCommand.execute(CompileCommand.java:1)
	at edu.kit.pp.mjcompiler.Main.lambda$0(Main.java:21)
	at edu.kit.pp.mjcompiler.Main$$Lambda$1/33524623.accept(Unknown Source)
	at java.util.Optional.ifPresent(Optional.java:159)
	at edu.kit.pp.mjcompiler.Main.main(Main.java:16)
./compareAll: line 19: ./Simon_150.s: No such file or directory
```

**Comment by felixwilhelm** [comment]
Hmm. Kümmer ich mich drum.

**Comment by felixwilhelm** [comment]
Fixed with d1f97c1e835ea5660fe986be8410d124f1e3fda8
